cdap-docs/build.sh

```diff
@@ -182,24 +188,43 @@ function build_javadocs() {
     local debug_flag=''
   fi
   if [[ ${javadoc_type} == ${ALL} ]]; then
-    javadoc_run="javadoc:aggregate"
+    javadoc_run="javadoc:aggregate -P release"
   else
-    javadoc_run="site"
+    javadoc_run="site -P templates"
```

Comment:
Why are we changing how this worked? We're mixing commands and profiles. If this is necessary, split these into separate variables... better to leave them as they were with the other changes in these comments.